### PR TITLE
3316 Updated guidance for international phone numbers

### PR DESCRIPTION
**Pull request by babsdenney**
- Added guidance for international phone numbers
- Added screenshots for examples
- Updated the phone-numbers.csv with international phone number labels
- Updated link to phonePatterns.jsx

### CODE REVIEW

File: src/_patterns/ask-users-for/phone-numbers.md [hunk 1]
```diff
@@ -39,9 +44,16 @@ anchors:
...
+* **Validate phone numbers.** Users must provide at least a 10 digit phone number with or without dashes for U.S. phone numbers and up to a 15-digit phone number for international phone numbers. 
+
+Example of acceptable formats for U.S. phone numbers:
   * 703-123-4567
   * 7031234567
+
+Example of acceptable formats for international phone numbers:
+  * +1-703-123-4567
+  * 17031234567
+
```
Comment (humancompanion-usds) [inline src/_patterns/ask-users-for/phone-numbers.md:56]: Can we add some non-U.S. examples here?

**Review by danbrady — APPROVED** [review]

**Comment by megzehn** [comment]
Thanks for adding this @babsdenney! I took a look since Laura has a lot on her plate this week. 

Does it mess things up if I suggest a small tweak to the error message in the [content considerations CSV](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/edit/3316-Add-international-phone-number-guidance-to-phone-number-pattern/src/_data/content/patterns/ask-users-for/phone-numbers.csv?pr=%2Fdepartment-of-veterans-affairs%2Fvets-design-system-documentation%2Fpull%2F3422)? I think this is a little clearer:

> Enter a phone number with up to 15 digits

But I realize we already have a screenshot. So I don't know how easy it is to tweak at this point.

**Comment by babsdenney** [comment]
@megzehn Thank you for looking over this! We can make the change, that will be no problem at all. I'll submit a new PR and add you as a reviewer.

**Comment by babsdenney** [comment]
@megzehn I added you to a PR for this update. I will delete that [screenshot that shows the error message](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/blob/main/src/images/patterns/ask-users-for/phone-numbers/international-phone-number-error.png). We aren't using that in the guidance, so that can be removed instead of updated.

**Comment by megzehn** [comment]
Thank you @babsdenney!! That change looks good.

**Pull request merged**